care/facility/migrations/0100_auto_20200418_2315.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 2.2.11 on 2020-04-18 17:45
+
+import care.utils.models.jsonfield
+from django.db import migrations, models
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [third party import "from django.db import migrations, models" should be placed before "import care.utils.models.jsonfield"](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5380979)